addons/pos_sale/tests/test_pos_sale_flow.py

```diff
@@ -873,14 +874,15 @@ def test_settle_order_ship_later_delivered_qty(self):
                 'price_unit': product_a.lst_price,
             })],
         })
-        sale_order.action_confirm()
 
         self.assertEqual(sale_order.order_line[0].qty_delivered, 0)
 
         self.main_pos_config.ship_later = True
         self.main_pos_config.open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'PosSettleOrderShipLater', login="accountman")
 
+        self.assertTrue(sale_order.picking_ids.state, 'cancel')
+
```

Comment:
```suggestion
        self.assertEqual(sale_order.picking_ids.state, 'cancel')

```

or 
```suggestion
        self.assertTrue(sale_order.picking_ids.state == 'cancel')

```
Otherwise the test doesn't fail.

Reply:
ah thanks for the catch ^^ mb :)